src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -21,7 +21,9 @@ except according to the terms contained in the LICENSE file.
         <expandable-row v-for="(form) in propertiesByForm" :key="form.xmlFormId">
           <template #title>
             <div class="form-name">
-              <router-link :to="publishedFormPath(projectId, form.xmlFormId)" v-tooltip.text>{{ form.name }}</router-link>
+              <form-link :form="form"
+                :to="publishedFormPath(form.projectId, form.xmlFormId)"
```

Comment:
We need to specify `:to` in this case. `FormLink` defaults to `primaryFormPath()`, but `primaryFormPath()` doesn't work super well in this case. That's because it expects the form object to have a `publishedAt` property.

Reply:
Just as projectId is set, we could also set publishedAt to something not null, but this is also fine.

Reply:
I agree that that would have been a reasonable approach as well. :+1: